audioswitch/src/main/java/com/twilio/audioswitch/android/BlacklistedDevices.kt

```diff
@@ -0,0 +1,80 @@
+package com.twilio.audioswitch.android
+
+internal val blacklistedDevices = listOf(
+    // Samsung Galaxy s9
+    "SM-G960W",
```

Comment:
Lets change the name so its specific to what feature we are working around/excluding & use regular expressions such that if samsung adds a new device model/color, we don't need to keep updating the list.

Reply:
I removed the device list all together. I think the comment above the code explains which devices are included. What you think